content/en/platform/corda/5.0/reference/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "References"
+project: corda
+version: 'Corda 5.0'
+date: 2023-04-24
+menu:
+  corda5:
+    identifier: corda5-references
+    weight: 6000
+section_menu: corda5
+---
+This section contains reference material for use by multiple Corda personas. It contains the following:
+* [Configuration Fields]({{< relref "./config-fields/_index.md" >}})
```

Comment:
why not use {{childpages}} here too ?

Reply:
The REST API page is a HTML file that's generated from the code. It doesn't contain Hugo front matter to use to sort the list of child pages.